doc/json-schema-type.md

```diff
@@ -71,14 +71,15 @@ Alternatively:
 }
 ```
 
-But wait, there is more!  You can optionally specify `"usage"` or `"image"` fields.
-The `"usage"` can be used to explain how the search works, i.e. if your feed
-supports special structures such as: `id:123`.  While the `"image"` fields are used
-add an image to the result.  For example:
+But wait, there is more!  You can also specify `"info": "Lorem ipsum"` to add a non-selectable
+item in the results list.  This could, for example, be used to explain how the search works,
+i.e. if your feed supports special structures such as: `id:123`.  While the `"image"` fields
+are used add an image to the result.  For example:
 ```json
 {
-    "usage": "Remember that you can also search for dates using: 'date:2020-07-24'",
+    "usage": "",
```

Comment:
Het `usage` veld zou hier dan toch uit moeten? Deze wordt niet meer gebruikt in de `switch` hieronder

Reply:
Oh klopt.  Ga ik aanpassen